frontend_server/lib/server.dart

```diff
@@ -22,6 +22,7 @@ import 'package:kernel/binary/limited_ast_to_binary.dart';
 import 'package:kernel/kernel.dart' show Program, loadProgramFromBytes;
 import 'package:kernel/target/flutter.dart';
 import 'package:kernel/target/targets.dart';
+import 'package:path/path.dart' as p;
```

Comment:
is it common to use 1 letter prefixes in Flutter code? 

maybe use `path` instead

Reply:
Not sure how common it is. I copied it from dart sdk where pub and analyzer seem to use 'p' as prefix.